core/amber/src/main/python/core/models/operator.py

```diff
@@ -229,9 +258,3 @@ def process_tuple(
             time, or None.
         """
         yield
-
-    def on_finish(self, port: int) -> Iterator[Optional[TupleLike]]:
-        """
-        For backward compatibility.
-        """
-        yield from self.process_tuple(InputExhausted(), input_=port)
```

Comment:
major: what happens to existing UDFs that are overriding this on_finish? will removing this cause backward compatibility issue? Notice that this function has `For backward compatibility.` in comments.

Reply:
Python operator V1 is removed.